codegen/runner/pre-steps.sh

```diff
@@ -190,14 +182,6 @@ for config_file in ${config_files}; do
 done
 target_dirs=($(echo "$target_dirs" | tr ' ' '\n' | sort | uniq))
 
-echo "Generating JSON Marshal/Unmarshal"
```

Comment:
what about the block above starting from line# 143.

Reply:
Yup, not-needed. Removed